jujucrashdump/crashdump.py

```diff
@@ -339,6 +342,24 @@ def retrieve_unit_tarballs(self):
             [(self.uniq, key, value, all_machines) for key, value in aliases.items()],
         )
 
+    def get_caas_stuff(self):
+        juju_status = self.status
+        if juju_status["model"]["type"] != "caas":
+            return
+
+        if "KUBECONFIG" not in os.environ:
+            return
+
+        try:
+            subprocess.check_output(["which", "kubectl"])
+        except subprocess.CalledProcessError:
+            return
+
+        run_cmd(
```

Comment:
no fstring?

Reply:
well.. we don't really have a 3.6+ policy for this project. we could adopt one I guess.